framework/db/Query.php

```diff
@@ -597,6 +597,35 @@ public function addSelect($columns)
     }
 
     /**
+     * Add columns of $realTable table to the SELECT part of the query, except $exceptColumns
+     *
+     * ```php
+     * $query->exceptSelect(["crowd", "crowdfunding", "title, content")->one();
+     * $query->exceptSelect(["crowd", "crowdfunding", ["title", "content"])->one();
+     * ```
+     *
+     * @param string $asTable a table name will be rename use "as" in SQL
+     * @param string $realTable a real table name
+     * @param string|array $exceptColumns columns will be remove from SELECT
+     * @return $this the query object itself
+     * @throws InvalidConfigException
+     */
+    public function exceptSelect($asTable, $realTable, $exceptColumns)
+    {
+        $tableSchema = Yii::$app->db->getSchema()->getTableSchema($realTable);
+        if ($tableSchema === null) {
+            throw new InvalidConfigException('The table does not exist: ' . $realTable);
+        }
+        $col = $tableSchema->getColumnNames();
+        $exceptColumns = is_array($exceptColumns) ? $exceptColumns : preg_split('/\s*,\s*/', trim($exceptColumns), -1, PREG_SPLIT_NO_EMPTY);
+        foreach($exceptColumns as $v) {
```

Comment:
`$v` is too short (codeclimate), we can use `$value` instead

Reply:
Ok,thank you to review the code,i will rename variables tomorrow